part-spec/common/thermals.json

```diff
@@ -0,0 +1,81 @@
+{
+    "$id": "https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/common/thermals.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "specification of thermals ",
+    "thermalSpec": {
+        "type": "object",
+        "properties": {
+            "junctionTemp": {
+                "description": "recommended operating junction temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "junctionTempMax": {
+                "description": "absolute maximum junction temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "ambientTemp": {
+                "description": "recommended operating ambient air temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "ambientTempMax": {
+                "description": "absolute maximum ambient air temperature range under which a part operates",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "caseTemp": {
+                "description": "case temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "caseTempMax": {
+                "description": "absolute maximum case temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "leadTempMax": {
+                "description": "absolute maximum lead temperature of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "storageTempMax": {
+                "description": "absolute maximum storage temperature range of a part",
```

Comment:
Sometimes you've used range for max and sometimes you just used temperature. I think we should pick one and be consistent.

I think "range" is fine for any of the non-max temperatures.